script.js

```diff
@@ -9,80 +9,71 @@ document.addEventListener("DOMContentLoaded", () => {
     }
   };
 
-  const createEl = (el, text, styles) => {
-    const tag = document.createElement(el);
-    tag.textContent = text;
-    tag.style.cssText = styles;
+  const createEl = (obj) => {
```

Comment:
Don't name entities with their type. Name it like args, config etc...
Use destructuring.